config.json

```diff
@@ -15,6 +15,14 @@
                 "file_uploads": true,
                 "file_actions": true,
                 "aws_actions": true
+            },
+            "share_notifications": {
+                "email": {
+                    "active": true,
```

Comment:
Should we default this to `false` as it requires `custom_domain` or `email_sender_id` in `cdk.json` which is not mandatory parameters?

Reply:
Good point! Making that change now, I don't want customers to start having unexpected issues out of nowwhere